.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+**Link the Issue(s) this Pull Request is related to.**
+
+Each PR should link at least one issue, in the form:
+
+```Fixes #123```
+
+Use one line for each Issue. This allows auto-closing the related issue when the fix is merged.
+
+**Summarize your change.**
+
+Describe the reason for the change.
+
+Add a list of changes, and note any that might need special attention during review.
+
+**Reference associated tests.**
+
+If no new tests are introduced as part of this PR, note the tests that are providing coverage.
+
+<!--
+Important: If this is your first contribution to OpenTimelineIO, you will need to submit a Contributor License Agreement. For a step-by-step instructions on the pull request process, see
+https://github.com/AcademySoftwareFoundation/OpenTimelineIO/tree/main/CONTRIBUTING.md
+-->
```

Comment:
Should this stay here? Will CLA/CCLA be required for the adapters repos?

Reply:
It is my assumption that it is needed for officially supported adapters.

Reply:
We discussed the pros/cons among the TSC and decided that yes, we would like to have the same CLA (and DCO) requirements for repos in the OpenTimelineIO GitHub organization. That will ensure that all the approvals and such are uniform between these and the core repo in the ASWF org. If a particular repo needs/wants to follow different rules, it can just live in a different org.
Does that sound reasonable to you two @markreidvfx and @JeanChristopheMorinPerso ?

Reply:
Sounds reasonable to me!
I don't think DCO has been activated for this repo yet, I believe its somewhere in the repo settings. 
If possible having `EasyCLA` available for plugin repos at some point too would be great too.

Reply:
I have no problem with DCO and CLAs in general, but I do find that they are kind of a barrier of entry in some cases. I find that it's a lot of bureaucracy to contribute to an adapter. But I guess there is some legal questions regarding adapters that are moved out of the OTIO repo and stuff like that? Anyway, if the TSC decided to go with CLS+DCO, then that's the way it is. I'll still happily sign the CLAs and contribute :)